java/ql/src/semmle/code/java/dataflow/internal/rangeanalysis/SignAnalysisSpecific.qll

```diff
@@ -69,8 +79,15 @@ private module Impl {
     e.(MethodAccess).getMethod() instanceof MapSizeMethod
   }
 
+  /** Holds if `e` is by definition strictly positive. */
   predicate positiveExpression(Expr e) { none() }
 
+  /**
+   * Holds if `e` has type `NumericOrCharType`, but the sign of `e` is unknown.
+   *
+   * The expression types handled in the predicate complements the expression
+   * types handled in `specificSubExprSign`.
+   */
   predicate unknownIntegerAccess(Expr e) {
```

Comment:
This predicate still needs a new name. It is not an access to an unknown integer nor an integer access that is unknown, so both possible parses of the name yields wrong meanings.

Reply:
How about `numericExprWithUnknownSign`?

Reply:
That sounds fine.